packages/components/Menu/src/MenuPopover/useMenuPopover.ts

```diff
@@ -101,7 +101,8 @@ export const useMenuPopover = (props: MenuPopoverProps): MenuPopoverState => {
     props: {
       accessibilityRole,
       target: triggerRef,
-      onDismiss,
+      onDismiss: props.onDismiss ?? onDismiss,
```

Comment:
Can we override our local `onDismiss` to call `props.onDismiss if available?

Reply:
Speaking of... do we need to do a similar thing for `onShow`? 

AKA, onDismiss is currently defined as
>const onDismiss = React.useCallback(() => setOpen(undefined, false /* isOpen */), [setOpen]);

should onShow be:

>const onShow = React.useCallback(() => setOpen(undefined, **true** /* isOpen */), [setOpen]);

Reply:
Edit: I think the answer to that is no because setOpen(true) is done by some timers...